diffsync/__init__.py

```diff
@@ -663,6 +714,34 @@ def get_by_uids(
         """
         return self.store.get_by_uids(uids=uids, model=obj)
 
+    def get_tree_traversal(self, as_dict: bool = False) -> Any:
```

Comment:
Could this be made into a `classmethod` so that you don't have to instantiate the adapter before being able to get this information?

Reply:
made update, using `cls.__name__` as the first line now.